Packages/io.chainsafe.web3-unity/Runtime/Scripts/EVM/Token/Erc20.cs

```diff
@@ -1,40 +1,65 @@
 using System.Numerics;
 using System.Threading.Tasks;
-using ChainSafe.Gaming.Web3;
+using ChainSafe.Gaming.Evm.Providers;
+using ChainSafe.Gaming.UnityPackage;
 
 namespace Scripts.EVM.Token
 {
-    // todo convert this into a service
     public class Erc20
     {
-
         private static readonly string Abi = ABI.Erc20;
+
         /// <summary>
         /// Balance Of ERC20 Address
         /// </summary>
         /// <param name="web3"></param>
         /// <param name="contractAddress"></param>
         /// <param name="account"></param>
         /// <returns></returns>
-        public static async Task<BigInteger> BalanceOf(Web3 web3, string contractAddress, string account)
+        public static async Task<BigInteger> BalanceOf(string contractAddress, string account)
         {
-            var contract = web3.ContractBuilder.Build(Abi, contractAddress);
+            var contract = Web3Accessor.Web3.ContractBuilder.Build(Abi, contractAddress);
```

Comment:
If this class isn't going to be part of Samples we should not use `Web3Accessor`

Reply:
noted, updated